lblod/frontend-form-builder
# SFB-137: When routing away from the edit route show save modal

## ID
 [SFB-137](https://binnenland.atlassian.net/browse/SFB-137)

 ## Description

When routing away from a `formbuilder/edit` route when the user has unsaved changes show a modal where the user can save or discard it's changes.

 ## Type of change


 ## How to test

1. open a form
2. Route to `codelijsten` ( top right)
3. did you see a modal? NO
4. Go back to your form and change the title of the section
5. Go to codelijsten
6. Did you see a modal? YES
7. Try the discard first and than the save unsaved changes
 
## Links to other PR's

 - /
